lib/ansiblelint/errors.py

```diff
@@ -17,3 +17,21 @@ def __repr__(self):
         formatstr = u"[{0}] ({1}) matched {2}:{3} {4}"
         return formatstr.format(self.rule.id, self.message,
                                 self.filename, self.linenumber, self.line)
+
+
+class MatchError(Exception):
```

Comment:
Would it be semantically better to inherit something a bit more concrete?
```suggestion
class MatchError(ValueError):
```